src/modules/coreHttpApi/debug-controllers/IdentityDeletionController.ts

```diff
@@ -0,0 +1,20 @@
+import { TransportServices } from "@nmshd/runtime";
+import { Inject } from "@nmshd/typescript-ioc";
+import { Accept, DELETE, Path } from "@nmshd/typescript-rest";
+import { Envelope } from "../../../infrastructure";
+import { BaseController } from "../common/BaseController";
+
+@Path("/api/v2/IdentityDeletion")
+export class IdentityDeletionController extends BaseController {
+    public constructor(@Inject private readonly transportServices: TransportServices) {
+        super();
+    }
+
+    @DELETE
```

Comment:
I see what you did here - "deleting" the identity using the delete keyword, but this is not REST conform.

1. rename everything here from IdentityDeletion to IdentityDeletionProcesses
2. make this a POST
3. rename this function please, and be more cautious with copy pasting code

Additionally this feature is non-sense if we don't allow to cancel / query the identity deletion process, please add this as routes.